HW_1/task_3.py

```diff
@@ -0,0 +1,19 @@
+# Напишите программу, которая принимает на вход координаты точки (X и Y) и выдаёт номер четверти плоскости,
+# в которой находится эта точка (или на какой оси она находится).
+
+x = int(input('Введите координату x = '))
+y = int(input('Введите координату y = '))
+if x == 0 and y == 0:
+    print('начало координат')
+elif x == 0 and y != 0:
+    print('ось X')
+elif y == 0 and x != 0:
+    print('ось Y')
+elif x > 0 and y > 0:
```

Comment:
Это и остальные условия можно свернуть относительно нуля.
```suggestion
elif x > 0 < y:
```